draft-ietf-quic-transport.md

```diff
@@ -1768,9 +1772,9 @@ completes, the client uses the transport parameters established in the
 handshake.  Not all transport parameters are remembered, as some do not apply to
 future connections or they have no effect on use of 0-RTT.
 
-The definition of new transport parameters ({{new-transport-parameters}}) MUST
-specify whether they MUST, MAY, or MUST NOT be stored for 0-RTT. A client need
-not store a transport parameter it cannot process.
+The definition of a new transport parameter ({{new-transport-parameters}}) is
+REQUIRED to specify whether it MUST, MAY, or MUST NOT be stored for 0-RTT. A
```

Comment:
This is fine, but I tend to prefer:

```suggestion
The definition of a new transport parameter ({{new-transport-parameters}}) MUST
specify whether that storing the transport parameter for 0-RTT is mandatory, optional, or prohibited. A
```

(Whether an extension uses 2119 language or not is not relevant, the level of requirement is.  And as these keywords are not used in this context in the way described in RFC 2119, this is better.)